diff -urN boringssl-orig/crypto/CMakeLists.txt boringssl/crypto/CMakeLists.txt
--- boringssl-orig/crypto/CMakeLists.txt	2019-11-08 14:59:15.000000000 +0300
+++ boringssl/crypto/CMakeLists.txt	2019-11-08 18:19:21.000000000 +0300
@@ -453,20 +453,6 @@
   target_link_libraries(crypto libcxx)
 endif()
 
-# urandom_test is a separate binary because it needs to be able to observe the
-# PRNG initialisation, which means that it can't have other tests running before
-# it does.
-add_executable(
-  urandom_test
-
-  fipsmodule/rand/urandom_test.cc
-)
-
-target_link_libraries(urandom_test test_support_lib boringssl_gtest crypto)
-
-add_dependencies(urandom_test global_target)
-add_dependencies(all_tests urandom_test)
-
 add_executable(
   crypto_test
 
