diff -urN boringssl-orig/ssl/s3_lib.cc boringssl/ssl/s3_lib.cc
--- boringssl-orig/ssl/s3_lib.cc	2017-11-27 23:19:17.297484878 +0100
+++ boringssl/ssl/s3_lib.cc	2017-11-27 23:20:24.000000000 +0100
@@ -199,7 +199,7 @@
   // TODO(davidben): Move this field into |s3|, have it store the normalized
   // protocol version, and implement this pre-negotiation quirk in |SSL_version|
   // at the API boundary rather than in internal state.
-  ssl->version = TLS1_2_VERSION;
+  ssl->version = TLS1_3_VERSION;
   return true;
 }
 
diff -urN boringssl-orig/ssl/ssl_versions.cc boringssl/ssl/ssl_versions.cc
--- boringssl-orig/ssl/ssl_versions.cc	2017-11-27 23:19:17.294484878 +0100
+++ boringssl/ssl/ssl_versions.cc	2017-11-27 23:21:15.000000000 +0100
@@ -209,7 +209,7 @@
                             uint16_t version) {
   // Zero is interpreted as the default maximum version.
   if (version == 0) {
-    *out = TLS1_2_VERSION;
+    *out = TLS1_3_VERSION;
     return true;
   }
 
