diff -urN incubator-pagespeed-ngx-1.12.34.3-stable-orig/config incubator-pagespeed-ngx-1.12.34.3-stable/config
--- incubator-pagespeed-ngx-1.12.34.3-stable-orig/config	2017-09-28 16:48:58.000000000 +0400
+++ incubator-pagespeed-ngx-1.12.34.3-stable/config	2018-09-18 13:59:50.000000000 +0400
@@ -76,7 +76,7 @@
 fi
 
 if [ "$NGX_DEBUG" = "YES" ]; then
-  buildtype=Debug
+  buildtype=Release
   # If we're using a psol tarball that doesn't contain Debug/ (which is the case
   # from 1.12 onward) then this will be overriden to buildtype=Release below.
 else
