diff -urN nginx-1.21.4-orig/conf/mime.types nginx-1.21.4-mime/conf/mime.types
--- nginx-1.21.4-orig/conf/mime.types	2021-11-02 17:49:22.000000000 +0300
+++ nginx-1.21.4-mime/conf/mime.types	2021-11-13 13:52:45.000000000 +0300
@@ -27,6 +27,8 @@
 
     font/woff                                        woff;
     font/woff2                                       woff2;
+    application/x-font-ttf                           ttf;
+    application/vnd.ms-opentype                      otf;
 
     application/java-archive                         jar war ear;
     application/json                                 json;
