diff -urN nginx-1.15.1-orig/conf/mime.types nginx-1.15.1/conf/mime.types
--- nginx-1.15.1-orig/conf/mime.types	2018-07-03 18:07:44.000000000 +0300
+++ nginx-1.15.1/conf/mime.types	2018-07-04 02:10:22.000000000 +0300
@@ -16,7 +16,7 @@
     text/x-component                                 htc;
 
     image/png                                        png;
-    image/svg+xml                                    svg svgz;
+    image/svg+xml                                    svg svgz xng;
     image/tiff                                       tif tiff;
     image/vnd.wap.wbmp                               wbmp;
     image/webp                                       webp;
@@ -24,6 +24,8 @@
     image/x-jng                                      jng;
     image/x-ms-bmp                                   bmp;
 
+    application/x-font-ttf                           ttf;
+    application/vnd.ms-opentype                      otf;
     font/woff                                        woff;
     font/woff2                                       woff2;
 
