diff -urN ngx_pagespeed-1.12.34.2-stable-orig/src/ngx_pagespeed.cc ngx_pagespeed-1.12.34.2-stable/src/ngx_pagespeed.cc
--- ngx_pagespeed-1.12.34.2-stable-orig/src/ngx_pagespeed.cc	2017-08-10 01:04:16.318960046 +0200
+++ ngx_pagespeed-1.12.34.2-stable/src/ngx_pagespeed.cc	2017-08-10 01:05:33.000000000 +0200
@@ -3008,8 +3008,7 @@
 
   i = r->phase_handler;
   // move handlers before try_files && content phase
-  while (ph[i + 1].checker != ngx_http_core_try_files_phase &&
-         ph[i + 1].checker != ngx_http_core_content_phase) {
+  while (ph[i + 1].checker != ngx_http_core_content_phase) {
     ph[i] = ph[i + 1];
     ph[i].next--;
     i++;
