diff -urN lua-nginx-module-0.10.15-orig/src/ngx_http_lua_module.c lua-nginx-module-0.10.15/src/ngx_http_lua_module.c
--- lua-nginx-module-0.10.15-orig/src/ngx_http_lua_module.c	2019-05-08 02:02:31.000000000 +0300
+++ lua-nginx-module-0.10.15/src/ngx_http_lua_module.c	2019-07-26 14:30:07.000000000 +0300
@@ -919,7 +919,7 @@
     ngx_http_lua_main_conf_t *lmcf = conf;
 
     if (lmcf->load_resty_core == NGX_CONF_UNSET) {
-        lmcf->load_resty_core = 1;
+        lmcf->load_resty_core = 0;
     }
 
 #if (NGX_PCRE)
